ci-automation/ci_automation_common.sh

```diff
@@ -212,11 +212,16 @@ function docker_image_from_buildcache() {
         return
     fi
 
+    # First try bincache then release to allow a bincache overwrite
     local url="https://${BUILDCACHE_SERVER}/containers/${version}/${tgz}"
+    local url_release="https://mirror.release.flatcar-linux.net/containers/"
```

Comment:
`url_release` should point to a file:

```suggestion
    local url_release="https://mirror.release.flatcar-linux.net/containers/${version}/${tgz}"
```